Week-5/Exercise-5.1/script.js

```diff
@@ -0,0 +1,27 @@
+const doTheTask = (taskNumber) => {
```

Comment:
I think this is not what assignment is asking. We need to create a custom function which uses both async/await and generators. The mixed function should attempt to call three task. Go over below link to understand better

https://www.pluralsight.com/guides/using-asyncawait-with-generator-functions